sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -50,10 +51,13 @@ class DynamoDBOnlineStoreConfig(FeastConfigBaseModel):
     """Online store type selector"""
 
     region: StrictStr
-    """ AWS Region Name """
+    """AWS Region Name"""
 
     table_name_template: StrictStr = "{project}.{table_name}"
-    """ DynamoDB table name template """
+    """DynamoDB table name template"""
+
+    sort_response: bool = True
+    """Whether or not to sort BatchGetItem response."""
```

Comment:
one final nit: i'd have the batch_size be a parameter here in teh online store config so that users can have some control. e.g. if they expect items to be very large, they may want to lower the batch size

Reply:
@adchia agree, user can specify the `batch_size` value defaulting to 40 in `DynamoDBOnlineStoreConfig`

```python
batch_size: int = 40
"""Number of items to retrieve in a DynamoDB BatchGetItem call."""
```